webpack.prod.config.ts

```diff
@@ -24,8 +24,13 @@ module.exports = {
   resolve: {
     modules: [
       path.resolve(__dirname, 'core/templates/dev/head'),
+      path.resolve(__dirname, 'node_modules'),
       path.resolve(__dirname, 'extensions')
     ],
+    alias: {
+      '@angular/upgrade/static': (
+        '@angular/upgrade/bundles/upgrade-static.umd.js')
+    }
```

Comment:
Can you explain this?

Reply:
This is one of the requirements. I faced errors on removing this.